Examples.fshx

```diff
@@ -53,32 +133,154 @@ Instance: mCODEKarnofskyPerformanceStatusExample01
 InstanceOf: KarnofskyPerformanceStatus
 * meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/KarnofskyPerformanceStatus"
 * status = #final "final"
+* category = #survey "survey"
 * method = SCT#5880005 "Clinical examination"
+* code = LNC#89245-5 "Karnofsky Performance Status [Interpretation]"
 * subject = Reference(mCODEPatientExample01)
 * effectiveDateTime = "2019-04-01"
 * performer = Reference(mCODEPractitionerExample01)
 * valueCodeableConcept = LNC#LA29177-5 "Normal activity with effort; some signs or symptoms of disease"
 
+
 Instance: mCODECancerRelatedMedicationStatementExample01
 InstanceOf: CancerRelatedMedicationStatement
 * meta.profile = "http://hl7.org/fhir/us/mcode/StructureDefinition/CancerRelatedMedicationStatement"
 * status = #active "active"
-* medicationCodeableConcept = RXN#93698 "Carboplatin Injectable Solution [Paraplatin]"
+* category = #community "community"
+* medicationCodeableConcept = RXN#349472 "gefitinib 250 MG Oral Tablet"
 * subject = Reference(mCODEPatientExample01)
 * effectiveDateTime = "2019-04-01"
 * dateAsserted = "2019-04-01"
-* extension.valueCodeableConcept = SCT#373808002 "Curative - procedure intent"
 * extension[TreatmentIntent].valueCodeableConcept = SCT#373808002 "Curative - procedure intent"
+* dosage.text = "250mg orally once daily with or without food"
+* dosage.route = SCT#26643006 "Oral use"
+* dosage.dose[x].system = UCUM#mg "mg"
+* dosage.dose[x].value = "250"
```

Comment:
You can't do a fixed value rule with an [x] choice. You have to specify the particular type the value should be applied to. You want to set doseQuantity here. You also missed the "doseAndRange" element in the path. So, I believe it should be:

* dosage.doseAndRate.doseQuantity.value = 250.0 // a decimal number, not a string
* dosage.doseAndRate.doseQuantity = UCUM#mg "mg"

The word "units" can be added to the latter expression to make it more clear what's happening, but it hasn't been implemented in SUSHI yet. In the future:

* dosage.doseAndRate.doseQuantity units = UCUM#mg "mg"